commands.json

```diff
@@ -0,0 +1 @@
+["calculate-checksum","changelog-sync","changelog-sync-sql","changelog-sync-to-tag","changelog-sync-to-tag-sql","checks bulk-set","checks copy","checks create","checks customize","checks delete","checks disable","checks enable","checks reset","checks run","checks show","clear-checksums","connect","db-doc","diff","diff-changelog","drop-all","execute-sql","flow","flow validate","future-rollback-count-sql","future-rollback-from-tag-sql","future-rollback-sql","generate-changelog","history","init copy","init project","init start-h2","list-locks","mark-next-changeset-ran","mark-next-changeset-ran-sql","release-locks","rollback","rollback-count","rollback-count-sql","rollback-one-changeset","rollback-one-changeset-sql","rollback-one-update","rollback-one-update-sql","rollback-sql","rollback-to-date","rollback-to-date-sql","set-contexts","set-labels","snapshot","snapshot-reference","status","tag","tag-exists","unexpected-changesets","update","update-count","update-count-sql","update-one-changeset","update-one-changeset-sql","update-sql","update-testing-rollback","update-to-tag","update-to-tag-sql","validate"]
```

Comment:
Hey @mcred I agree with you but Spacelift deploys the stuff which is only in the repository. Spacelift can only "user" local files for previewing. Spacelift can perform some kind of terraform planning using the current working directory, but it will only deploy pushed stuff. We could remove the file from the repo after executing the `spacectl stack deploy --id liquibase-github-actions --auto-confirm` command.

Reply:
"will only deploy pushed stuff" seems problematic. Would this mean that during automation we would need to update the file, push it to the repo, and then let spacelift check it out?

Reply:
You are already updating the file with https://github.com/liquibase/github-action-generator/blob/f6c751d05949576a3ce53c7be7b7d7f05490c7c5/.github/workflows/generate.yml#L18

The only change is about pushing the file if there are changes on it.

Reply:
Hey @mcred I think we can do the following to not push the file.

We can run the following `spacectl` command before deploying stuff:

```bash
spacectl stack environment mount [command options] RELATIVE_PATH_TO_MOUNT [FILE_PATH]
```

```bash
spacectl stack environment mount --id liquibase-github-actions commands.json commands.json 
```

That command will mount the `commands.json` file in the Spacelift backend ready to be used by Terraform:

![SCR-20240111-hezn](https://github.com/liquibase/github-action-generator/assets/67458478/7b0ff4f0-82a3-4f66-a408-a85dd48fb77f)

I have updated the PR with the change. Please tell me if this works better.